src/test/java/code401challenges/BinarySearchTest.java

```diff
@@ -0,0 +1,23 @@
+package code401challenges;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class BinarySearchTest {
+
+    @Test
+    public void main() {
+    }
+
+    @Test
+    public void searchTest() {
+        int expected = BinarySearch.search(new int[]{1, 2, 3, 4, 5, 6, 7, 8}, 8);
+        assertEquals(7, expected);
+    }

```

Comment:
This test will work, but I'm worried you are flipping the variable. With the assert() methods, it goes expected first, then acutal. Basically, expected on the left, and the method call on the right, which will give you the actual.